src/Reflection/Adapter/ReflectionEnum.php

```diff
@@ -376,6 +376,50 @@ public function newInstanceArgs(array|null $args = null): object
         throw new Exception\NotImplemented('Not implemented');
     }
 
+    /** @param int-mask-of<ReflectionClass::SKIP_*> $options */
+    public function newLazyGhost(callable $initializer, int $options = 0): object
+    {
+        throw new Exception\NotImplemented('Not implemented');
```

Comment:
Similar to above: I think we probably need something slightly better than `NotImplemented`, just to signal a design decision, rather than a "TODO"